test/integration/golden_response/get_statvar_group/uk.json

```diff
@@ -1358,6 +1358,211 @@
           "id": "Count_Product_MobileCellularSubscription_AsFractionOf_Count_Person",
           "searchName": "Count Of Product With Product Type Mobile Cellular Subscription As Fraction Of Count Person",
           "displayName": "Count Of MobileCellularSubscription As Fraction Of Count Person"
+        },
```

Comment:
wonder if these are new data added?

Reply:
there isn't an explicit data addition I see.  my guess is perhaps the covid mobility dataset started including uk stats?